### PR TITLE
Update devDependencies

### PR DESCRIPTION
**Pull request by JuliaGerasymenko**

### CODE REVIEW

**Comment by tshemsedinov** [comment]
Land after https://github.com/metarhia/metalog/pull/84

File: package.json [hunk 1]
```diff
@@ -33,10 +33,10 @@
     "metastreams": "^0.1.0"
   },
   "devDependencies": {
-    "eslint": "^5.13.0",
+    "eslint": "^5.15.3",
```
Comment (SemenchenkoVitaliy) [inline package.json]:
```suggestion
    "eslint": "^5.16.0",
```

File: package.json [hunk 2]
```diff
@@ -33,10 +33,10 @@
...
   },
   "devDependencies": {
-    "eslint": "^5.13.0",
-    "eslint-config-prettier": "^4.0.0",
+    "eslint": "^5.16.0",
+    "eslint-config-prettier": "^4.1.0",
     "eslint-plugin-prettier": "^3.0.1",
-    "metatests": "^0.4.1",
+    "metatests": "^0.6.5",
     "prettier": "^1.16.4"
```
Comment (belochub) [inline package.json]: There's `prettier@1.17.0` already, though it can be updated on landing.

**Comment by belochub** [comment]
@nechaido, land this, please.

**Pull request merged**